TLF-26 & TLF-38: Add summary page and reference number

mislam987 opened the pull request:
## What
- Merge branches together to form the final summary page
- Add summary page [TLF-26](https://collaboration.homeoffice.gov.uk/jira/browse/TLF-26)
- Generate reference number and add to email [TLF-38](https://collaboration.homeoffice.gov.uk/jira/browse/TLF-38)

## Why
- For the content to match the wireframe designs

## How
- Add all fields to /sections/summary-data-sections.js
- Add sections and fields to confirm page in pages.json
- Use UUID to generate the reference number in /behaviours/create-and-send-pdf.js
- Modify personalisations for email generation to use the session values in /behaviours/create-and-send-pdf.js

## Testing
- Tests passing locally

PaolaDMadd-Pro reviewed (approved):
- Asked to add reference number to confirmation page.

mislam987 commented:
As discussed previously, I've made the following changes that we agreed:
- Added reference number to confirmation page and PDF
- Resolved the double-click issue on the privacy page by adding a spinning animation to the submit button once it is clicked
- Changed PDF styling to have extra padding
